gradle/plugins/src/main/kotlin/com.hedera.gradle.java-base.gradle.kts

```diff
@@ -45,56 +42,26 @@ jvmDependencyConflicts {
     consistentResolution.platform("com.hedera.hashgraph:sdk-dependency-versions")
 }
 
-dependencies {
-    // https://github.com/google/error-prone
-    // https://errorprone.info/
-    errorprone("com.google.errorprone:error_prone_core:2.21.1")
-
-    // https://github.com/uber/NullAway
-    errorprone("com.uber.nullaway:nullaway:0.10.14")
-
-    // https://github.com/grpc/grpc-java-api-checker
-    errorprone("io.grpc:grpc-java-api-checker:1.1.0")
-}
+val deactivatedCompileLintOptions =
+    listOf(
+        "module", // module not found when doing 'exports to ...'
+        "serial", // serializable class ... has no definition of serialVersionUID
+        "processing", // No processor claimed any of these annotations: ...
+        "try", // auto-closeable resource ignore is never referenced... (AutoClosableLock)
+        "missing-explicit-ctor", // class ... declares no explicit constructors
+        "removal",
+        "deprecation",
+        "overrides", // overrides equals, but neither it ... overrides hashCode method
+        "unchecked",
+        "rawtypes",
+        "exports",
+        "dep-ann"
+    )
```

Comment:
This is what we also have in `hedera-services` and which will be provided by the shared configuration later.